sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -217,6 +218,27 @@ public String getAccountName() {
         return this.accountName;
     }
 
+    /**
+     * Get an async client pointing to the account.
+     *
+     * @return {@link BlobServiceAsyncClient}
+     */
+    public BlobServiceAsyncClient getServiceAsyncClient() {
```

Comment:
Should this get a code sample too?

Reply:
We don't have code samples for other getters on this type, but I can add some if we want